<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryPage</class>
 <widget class="QWidget" name="QueryPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextEdit" name="queryTextEdit"/>
   </item>
   <item>
    <widget class="QPushButton" name="executeButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>実行</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="queryResultView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
